src/distinguished_name.cpp

```diff
@@ -84,6 +90,11 @@ DistinguishedName DistinguishedName::fromX509Name(X509_NAME *x509)
     return builder.build();
 }
 
+const std::string& DistinguishedName::getSubjectAttributeByNIDAsString(const ASN1_NID id) const
+{
+    return _getSubjectAttributeByNID(id).name;
+}
+
 auto _createTuple(const DistinguishedName &dn)
 {
     return std::tie(dn.commonName(),
```

Comment:
This seems to be broken since the introduction of givenName(). Should be added here as well.

Reply:
We thought it is on purpose left out. I can added no problem.